benchmarks_runner/eval.py

```diff
@@ -90,8 +90,8 @@ def evaluate_results(
                 message = results_dir.get(uid, {}).get("message", {})
 
         results_k = sorted(
-            message.get('results', []),
-            key=lambda r: r['analyses'][0].get('score', 0),
+            message.get('results') or [],
```

Comment:
Was it intentional to change `message.get('results', [])` to `message.get('results') or []`?

We should avoid using `None` as a `bool`.

Reply:
Yes, this was intentional. There are cases when `results` gets sent as None and we need it to be a list in order for the sorted function to work. I was getting `Nonetype is not iterable` errors. i.e.
```json
{
  "message": {
    "query_graph": {...},
    "knowledge_graph": {},
    "results": null
  }
}
```